colors/build.js

```diff
@@ -25,11 +38,16 @@ const sassOutput = [`/* Firefox Colors SCSS Variables v${metadata.version} */
 
 `];
 
+const jsOutput = [`/* Firefox Colors JS v${metadata.version} */
```

Comment:
`JS Variables` just to match the other headers? 🙂

Reply:
I pushed on a commit tweaking this.